Samples/Samples/Samples/Views/EffectsEntryDisableAutoCorrectPage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace FormsCommunityToolkit.Samples.Views
+{
+	[XamlCompilation(XamlCompilationOptions.Compile)]
```

Comment:
Any reason why we can't move this to the assemblyinfo.cs an turn it on for the whole project?